src/org-folding-provider.ts

```diff
@@ -19,7 +19,7 @@ export class OrgFoldingProvider implements FoldingRangeProvider {
             const element = document.lineAt(lineNumber);
             const text = element.text;
 
-            if (!text.startsWith("*")) {
+            if (!text.match(/^\*+ /)) {
```

Comment:
It makes sense to use `/^\*+ /.test(text)` here as it's faster and the result of the `match()` call is not used anyway.

Reply:
Yes, I fixed it in my [fork](https://github.com/sipi/vscode-org-mode/commit/2d6093687f0d84b30d0e5f42219ee0e1c59456bc).